release-notes/7.0/preview/api-diff/preview1/Microsoft.WindowsDesktop.App/7.0-preview1_System.Media.md

```diff
@@ -0,0 +1,11 @@
+# System.Media
+
+``` diff
+ namespace System.Media {
+     public class SoundPlayer : Component, ISerializable {
+-        void System.Runtime.Serialization.ISerializable.GetObjectData(SerializationInfo info, StreamingContext context);
++        void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context);
```

Comment:
This one is strange. Unlike the other comment, where the issue was that the actual ref file had incomplete namespaces, in this case, the ref file shows that the API has its full namespace.

Git blame shows the last time this was modified was 3 years ago, so it might be a problem with the tool: https://github.com/dotnet/runtime/blame/89e3040713c9cc63c7e4922a0bbe7ff8456168bf/src/libraries/System.Windows.Extensions/ref/System.Windows.Extensions.cs#L33

I'll remove the file from this PR.